vllm/model_executor/layers/quantized_linear/gptq.py

```diff
@@ -0,0 +1,265 @@
+from typing import Optional
+
+import torch
+from torch.nn.parameter import Parameter
+
+from vllm import quantization_ops
+from vllm.model_executor.parallel_utils.tensor_parallel.layers import (
```

Comment:
There is an error when i run examples/offline_inference.py:

```
Traceback (most recent call last):
  File "examples/offline_inference.py", line 14, in <module>
    llm = LLM(model="/wangjiancheng/2023_09_08_14_59_40", trust_remote_code=True)
  File "/wangjiancheng/temp/vllm-gptq/vllm/entrypoints/llm.py", line 94, in __init__
    self.llm_engine = LLMEngine.from_engine_args(engine_args)
  File "/wangjiancheng/temp/vllm-gptq/vllm/engine/llm_engine.py", line 231, in from_engine_args
    engine = cls(*engine_configs,
  File "/wangjiancheng/temp/vllm-gptq/vllm/engine/llm_engine.py", line 110, in __init__
    self._init_workers(distributed_init_method)
  File "/wangjiancheng/temp/vllm-gptq/vllm/engine/llm_engine.py", line 128, in _init_workers
    from vllm.worker.worker import Worker  # pylint: disable=import-outside-toplevel
  File "/wangjiancheng/temp/vllm-gptq/vllm/worker/worker.py", line 10, in <module>
    from vllm.model_executor import get_model, InputMetadata, set_random_seed
  File "/wangjiancheng/temp/vllm-gptq/vllm/model_executor/__init__.py", line 2, in <module>
    from vllm.model_executor.model_loader import get_model
  File "/wangjiancheng/temp/vllm-gptq/vllm/model_executor/model_loader.py", line 10, in <module>
    from vllm.model_executor.models import *  # pylint: disable=wildcard-import
  File "/wangjiancheng/temp/vllm-gptq/vllm/model_executor/models/__init__.py", line 1, in <module>
    from vllm.model_executor.models.aquila import AquilaForCausalLM
  File "/wangjiancheng/temp/vllm-gptq/vllm/model_executor/models/aquila.py", line 37, in <module>
    from vllm.model_executor.layers.quantized_linear import ParallelLinear
  File "/wangjiancheng/temp/vllm-gptq/vllm/model_executor/layers/quantized_linear/__init__.py", line 5, in <module>
    from vllm.model_executor.layers.quantized_linear.gptq import (
  File "/wangjiancheng/temp/vllm-gptq/vllm/model_executor/layers/quantized_linear/gptq.py", line 7, in <module>
    from vllm.model_executor.parallel_utils.tensor_parallel.layers import (
ModuleNotFoundError: No module named 'vllm.model_executor.parallel_utils.tensor_parallel'

```

Then i change `vllm.model_executor.parallel_utils.layers` to work aroud. Is there an error here?

Reply:
You're right. The structure was changed recently in main branch and I forgot to change here while merging it.